public/src/Player.js

```diff
@@ -0,0 +1,43 @@
+function Player(keyboarder = new Keyboarder()) {
+  this.center = { x: 20, y: 0 };
+  this.size = { x: 20, y: 55 };
+  this.velocity = { x: 0, y: 0 };
+  this.movement = { x: 1.5, y: -25 };
+  this.keyboarder = keyboarder;
+}
+
+Player.prototype = {
+  update() {
+    this._moveX();
+    this._moveY();
+    this._movePlayer();
+  },
+
+  draw(context) {
+    context.fillRect(
+      this.center.x - (this.size.x / 2),
+      this.center.y - (this.size.y / 2),
+      this.size.x,
+      this.size.y
+    );
+  },
+
+  _moveX() {
+    if (this.keyboarder.isRightKeyDown()) {
```

Comment:
This if statement should set velocity to 1.5 if right pressed, -1.5 is left pressed and 0 in an else